pom.xml

```diff
@@ -135,6 +135,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.kiwiproject</groupId>
```

Comment:
Since kiwi brings in guava as a _required_ dependency, should we remove the guava dependency?

Reply:
I need it for exclusions

Reply:
Turns out, we don't need it, and it has been removed.

Reply:
nevermind